src/app/plugins/platform/Docker/plugins/cms/Drupal8/installDrupal.ts

```diff
@@ -14,10 +14,19 @@ export type PantheonProject = typeof pantheonProject;
 
 export type Project = PantheonProject | DrupalProject;
 
+// Promisify our utility methods.
 const readFile = promisify(fs.readFile);
 const writeFile = promisify(fs.writeFile);
 const rimrafAsync = promisify(rimraf);
 
+/**
+ * Replace values into a given file.
+ *
+ * @param {string} path
```

Comment:
TypeDoc doesn't use these annotations - these are only for JSDoc, which doesn't have a type system to draw this information from. If we don't have documentation for a specific parameter, we can leave out the entire tag.